### DISCUSSION TITLE
Provide example using vendor import map

### DISCUSSION THREAD

**Discussion by thegaryroberts** [freshframework/fresh#218] [Ideas]
I've updated my example project using the new import map approach (love it!).

As this project is now going to be unstable for a while longer, I attempted to use `deno vendor` to maintain a fixed local copy and minimise chance of breakage. Unfortunately I ran into this issue: https://github.com/denoland/deno/issues/13697

Is it possible an example could be documented that shows how to combine the vendor import map & the default project import map you have in place?

**Comment by lucacasonato** [comment]
Yeah, this is something that I'll definitely do soon. For now you can also lock the `fresh` version to a specific commit (eg `https://raw.githubusercontent.com/lucacasonato/fresh/3ff689d8f991b2aba7f03f62fd6b47705a7665df/server.ts`) instead of importing from `main`.

**Comment by PhoneDroid** [comment]
@marvinhagemeister 
Should be resolved, can be closed.

( Issue https://github.com/denoland/deno/issues/13697 had been resolved via https://github.com/denoland/deno/pull/14836 )
